src/cpu/reorder/cpu_reorder.cpp

```diff
@@ -40,6 +40,7 @@ regular_impl_list_map() {
             {{f4_e3m0, data_type::undef, 0}, &regular_fp4_impl_list_map()},
             {{f8_e5m2, data_type::undef, 0}, &regular_fp8_impl_list_map()},
             {{f8_e4m3, data_type::undef, 0}, &regular_fp8_impl_list_map()},
+            {{e8m0, f32, 0}, &regular_fp8_impl_list_map()},
```

Comment:
Any specific reason to have just f32 and not any data type instead?

Reply:
`e8m0 -> f32` is needed for testing (checking computed scales).
If I was to add others, would need to add ci coverage, and I do not know of use cases for those other flavors.